pkg/storaged/crypto-keyslots.jsx

```diff
@@ -432,7 +433,7 @@ const RemoveClevisField = (tag, key, dev) => {
 
 function remove_clevis_dialog(client, block, key) {
     dialog_open({
-        Title: _("Remove Tang keyserver"),
+        Title: <><ExclamationTriangleIcon className="ct-icon-exclamation-triangle" /> {_("Remove Tang keyserver")}</>,
```

Comment:
To make modals with warnings you are supposed to pass the titleIconVariant="warning" to the Modal component directly.
In this case you need to add the a new property to the Dialog in cockpit-components-dialog. 

In anycase, we can do it in followup.

Reply:
Yeah, I wanted to only focus on the body of dialogs in this PR as there are a lot of things we should do with modals themselves - like this alert, size of them, spinner position when action is happening...